src/components/views/dialogs/InviteDialog.js

```diff
@@ -332,7 +333,23 @@ export default class InviteDialog extends React.PureComponent {
     }
 
     _buildRecents(excludedTargetIds: Set<string>): {userId: string, user: RoomMember, lastActive: number} {
-        const rooms = DMRoomMap.shared().getUniqueRoomsWithIndividuals();
+        const rooms = DMRoomMap.shared().getUniqueRoomsWithIndividuals(); // map of userId => js-sdk Room
+
+        // Also pull in all the rooms tagged as m.direct so we don't miss anything. Sometimes the
```

Comment:
It seems confused to read "tagged as m.direct" when `TAG_DM` is something else...